### PR TITLE
uxDesign task

### PR DESCRIPTION
**Pull request by Heedharyhart**
I closed all my PR and taking the submission from the beginning due to alot of conflict my github is experiencing

### CODE REVIEW

**Comment by Heedharyhart** [comment]
these has not been merged

File: submissions/Abdulkareem Idayat Olaide [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (kenny-wealth) [inline submissions/Abdulkareem Idayat Olaide]: You may delete this file if not needed.

File: uxDesign [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (kenny-wealth) [inline uxDesign]: Delete this if not needed

**Review by kenny-wealth — CHANGES_REQUESTED** [review]

**Comment by Heedharyhart** [comment]
Pls, how do i delete it???? Have tried it, but is not working

**Comment by tegaphilip** [comment]
@Heedharyhart delete from your computer. then do a `git add`, `git commit -m 'message'` and a `git push origin master`

**Comment by Heedharyhart** [comment]
Alright
Thanks sir

**Pull request closed without merging**